tokio-quiche/src/quic/router/mod.rs

```diff
@@ -413,8 +415,6 @@ where
                 return self.poll_recv_from(cx);
             };
 
-            self.reusable_cmsg_space.clear();
```

Comment:
What was it doing and why dont we need it now?

Reply:
The previous version of mio built the vector based on the capacity of the passed buffer, while the new version does it based on length. clear() sets the length to 0, so using that ends up throwing ENOBUFS when we call cmsgs(). That said I did have a fill(0) here which must've gotten removed in a rebase, I'll put that back